src/eduid_userdb/userdb.py

```diff
@@ -288,3 +288,19 @@ def update_user(self, obj_id, attributes):
                         '$set': attributes,
                     }
                 )
+
+    def get_identity_proofing(self, user):
+        """
```

Comment:
I leave this as a method and noty as a user property, since it is used as a method.